### PR TITLE
adjust: RFC 5988 obsoleted by RFC 8288

### PR DESCRIPTION
**Pull request by fgeorgatos**
fyi:
* https://www.rfc-editor.org/info/rfc5988
* https://www.rfc-editor.org/info/rfc8288

### CODE REVIEW

File: docs/developer/contributing/api-guidelines.rst [hunk 1]
```diff
@@ -226,7 +226,7 @@ response indicating:
 - next page ``rel="next"``;
 - last page ``rel="last"``.
 
-Link headers are described by `RFC 5988
+Link headers are described by `RFC 8288
```
Comment (rokroskar) [inline docs/developer/contributing/api-guidelines.rst]: you should also change the actual link

Comment (fgeorgatos) [reply docs/developer/contributing/api-guidelines.rst]: @rokroskar, done: approve?

**Comment by fgeorgatos** [comment]
@ableuler : done, with thanks for the hint

File: docs/developer/contributing/api-guidelines.rst (hunk 1 above)
Comment (fgeorgatos) [reply docs/developer/contributing/api-guidelines.rst]: @ableuler : explicitly approve?

File: docs/developer/contributing/api-guidelines.rst [hunk 2]
```diff
@@ -226,8 +226,8 @@ response indicating:
 - next page ``rel="next"``;
 - last page ``rel="last"``.
 
-Link headers are described by `RFC 5988
-<https://tools.ietf.org/html/rfc5988#page-6>`_.
+Link headers are described by `RFC 8288
+<https://tools.ietf.org/html/rfc8288#page-6>`_.
```
Comment (ableuler) [inline docs/developer/contributing/api-guidelines.rst]: I see another remaining link to page 6 which is not of particular interest anymore.

Comment (fgeorgatos) [reply docs/developer/contributing/api-guidelines.rst]: @ableuler: fixed?

**Comment by fgeorgatos** [comment]
thanks @ableuler , squashed & merged.

**Pull request merged**